packages/extension-base/src/background/handlers/Tabs.ts

```diff
@@ -30,7 +30,7 @@ function transformAccounts (accounts: SubjectInfo, anyType = false): InjectedAcc
       address,
       genesisHash,
       name,
-      type
+      type: type || 'sr25519'
```

Comment:
I would rather leave this empty - for some we really don't know the types and this is probably incorrect. so for Ledger it is ed25519, for Signer it is sr25519, but neither is guaranteed. 

So it really should always be has something or we just don't know. (At least in the cases where it does matter, it is there, eg `ethereum`)